spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -30,6 +30,38 @@ Only this Specification will be generated and maintained to document the content
 
 In reverse chronological order, here's the specification for each MicroProfile release.
 
+[[microprofile2.1]]
+=== MicroProfile 2.1 === (4Q2018)
+MicroProfile 2.1 is the eigth release for the https://projects.eclipse.org/projects/technology.microprofile[Eclipse MicroProfile project].
+Based on MicroProfile's time-boxed release process, this is an incremental release that includes an update to OpenTracing 1.2.
+
+Thus, the complete list of functional components for MicroProfile 2.1 includes...
+
+ - https://github.com/eclipse/microprofile-config/releases/tag/1.3[MicroProfile Config 1.3]
+ - https://github.com/eclipse/microprofile-fault-tolerance/releases/tag/1.1[MicroProfile Fault Tolerance 1.1]
+ - https://github.com/eclipse/microprofile-health/releases/tag/1.0[MicroProfile Health Check 1.0]
+ - https://github.com/eclipse/microprofile-jwt-auth/releases/tag/1.1[MicroProfile JWT Authentication 1.1]
+ - https://github.com/eclipse/microprofile-metrics/releases/tag/1.1[MicroProfile Metrics 1.1]
+ - https://github.com/eclipse/microprofile-open-api/releases/tag/1.0[MicroProfile OpenAPI 1.0]
+ - https://github.com/eclipse/microprofile-opentracing/releases/tag/1.2[MicroProfile OpenTracing 1.2]
+ - https://github.com/eclipse/microprofile-rest-client/releases/tag/1.1[MicroProfile Rest Client 1.1]
+ - https://jcp.org/en/jsr/detail?id=365[CDI 2.0]
+ - https://jcp.org/en/jsr/detail?id=250[Common Annotations 1.3]
+ - https://jcp.org/en/jsr/detail?id=370[JAX-RS 2.1]
+ - https://jcp.org/en/jsr/detail?id=367[JSON-B 1.0]
+ - https://jcp.org/en/jsr/detail?id=374[JSON-P 1.1]
+
+The Maven coordinates for this Eclipse release are as follows:
+----
+<dependency>
+    <groupId>org.eclipse.microprofile</groupId>
+    <artifactId>microprofile</artifactId>
+    <version>2.1</version>
+    <type>pom</type>
+    <scope>provided</scope>
+</dependency>
+----
+
```

Comment:
You accidentally dropped the pointer to our github repo:  "Here is the link to the github repository for this Eclipse-based project."